starter-code/vue.js

```diff
@@ -0,0 +1,120 @@
+Vue.component('message', {
+	data: function() {
+		return { dumped: false }
+	},
+	props: {
+		header: String,
+		body: String,
+		color: {
+			type: String,
+			enum: ['dark', 'primary', 'link', 'info', 'success', 'warning', 'danger']
```

Comment:
You can't use enums with Vue. However you can use a validator, which is a function that will return whether you should validate the given prop value. You would implement it like this:

```javascript
props: {
    header: String,
    body: String,
    color: {
        validator(color) {
          return ['dark', 'primary', 'link', 'info', 'success', 'warning', 'danger'].includes(color)
        }
    }
}
```